tools/build-neatvnc.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Update and install dependencies
+sudo apt update
+sudo apt install -y meson ninja-build libpixman-1-dev zlib1g-dev libdrm-dev pkgconf
+
+CURRENT_DIR=$(pwd)
+# Clone and build aml
+git clone https://github.com/any1/aml && cd aml
```

Comment:
Replace `cd aml` with `pushd am` and then you can use `popd`.
See https://www.gnu.org/software/bash/manual/html_node/Directory-Stack-Builtins.html